solr/solrj/src/java/org/apache/solr/client/solrj/impl/ClusterStateProvider.java

```diff
@@ -31,7 +29,7 @@
 public interface ClusterStateProvider extends SolrCloseable {
 
   static ClusterStateProvider newZkClusterStateProvider(
-      Collection<String> zkHosts, String zkChroot, boolean canUseZkACLs) {
+          Collection<String> zkHosts, String zkChroot, boolean canUseZkACLs) {
```

Comment:
Run the Gradle tidy task to fix code formatting

Reply:
Done